src/css/flags/ads.less

```diff
@@ -38,6 +36,12 @@
 
 .jwplayer.jw-flag-ads-hide-controls {
     .jw-controls {
-      display : none !important;
+        display : none !important;
     }
 }
+
+.jwplayer.jw-flag-ads-mobile {
```

Comment:
I'd prefer a persistent mobile flag on the player. Then in this case you could use:
`.jwplayer.jw-flag-ads-hide-controls.jw-flag-mobile {`